src/Serilog.Sinks.Splunk/Sinks/Splunk/SplunkJsonFormatter.cs

```diff
@@ -10,56 +10,96 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.using System;
+// limitations under the License.
 
 using System;
+using System.Collections.Generic;
 using System.IO;
+using Serilog.Events;
+using Serilog.Formatting;
 using Serilog.Formatting.Json;
 
 namespace Serilog.Sinks.Splunk
 {
     /// <summary>
-    /// A json formatter to allow conditional rendering of properties
+    /// Renders log events into a default JSON format for consumption by Splunk.
     /// </summary>
-    public class SplunkJsonFormatter : JsonFormatter
+    public class SplunkJsonFormatter : ITextFormatter
     {
-        private readonly bool _renderTemplate;
+        static readonly JsonValueFormatter ValueFormatter = new JsonValueFormatter();
+
+        readonly bool _renderTemplate;
+        readonly bool _renderMessage;
+        readonly IFormatProvider _formatProvider;
 
         /// <summary>
-        /// Construct a <see cref="JsonFormatter"/>.
+        /// Construct a <see cref="SplunkJsonFormatter"/>.
         /// </summary>
-        /// <param name="omitEnclosingObject">If true, the properties of the event will be written to
-        /// the output without enclosing braces. Otherwise, if false, each event will be written as a well-formed
-        /// JSON object.</param>
-        /// <param name="closingDelimiter">A string that will be written after each log event is formatted.
-        /// If null, <see cref="Environment.NewLine"/> will be used. Ignored if <paramref name="omitEnclosingObject"/>
-        /// is true.</param>
         /// <param name="renderMessage">If true, the message will be rendered and written to the output as a
         /// property named RenderedMessage.</param>
         /// <param name="formatProvider">Supplies culture-specific formatting information, or null.</param>
         /// <param name="renderTemplate">If true, the template used will be rendered and written to the output as a property named MessageTemplate</param>
         public SplunkJsonFormatter(
-            bool omitEnclosingObject = false,
-            string closingDelimiter = null,
+            bool renderTemplate = true,
             bool renderMessage = false,
-            IFormatProvider formatProvider = null,
-            bool renderTemplate = true) 
-            :base(omitEnclosingObject,closingDelimiter,renderMessage,formatProvider)
+            IFormatProvider formatProvider = null)
         {
             _renderTemplate = renderTemplate;
+            _renderMessage = renderMessage;
+            _formatProvider = formatProvider;
         }
 
+        public void Format(LogEvent logEvent, TextWriter output)
```

Comment:
After a bit more digging, I can see the issue relates to the `renderTemplate`.

https://github.com/nblumhardt/serilog-sinks-splunk/blob/d6a5d9b881ca628c1eb6bd49474400fd9e3c0c8b/src/Serilog.Sinks.Splunk/Sinks/Splunk/SplunkJsonFormatter.cs#L62

No determination on the cause as yet.